### PR TITLE
Making logging uniform in VPA

### PR DESCRIPTION
**Pull request by Rajat-0**

### CODE REVIEW

File: vertical-pod-autoscaler/pkg/recommender/checkpoint/checkpoint_writer_test.go [hunk 1]
```diff
@@ -21,13 +21,13 @@ import (
...
 	"time"
 
-	"github.com/golang/glog"
 	"github.com/stretchr/testify/assert"
 	"k8s.io/api/core/v1"
 	apiv1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	vpa_types "k8s.io/autoscaler/vertical-pod-autoscaler/pkg/apis/autoscaling.k8s.io/v1beta2"
 	"k8s.io/autoscaler/vertical-pod-autoscaler/pkg/recommender/model"
+        "k8s.io/klog
```
Comment (bskiba) [inline vertical-pod-autoscaler/pkg/recommender/checkpoint/checkpoint_writer_test.go]: Missing " at the end. Please fix indent.

File: vertical-pod-autoscaler/pkg/target/composite_fetcher.go [hunk 2]
```diff
@@ -17,9 +17,9 @@ limitations under the License.
 package target
 
 import (
-	"github.com/golang/glog"
 	"k8s.io/apimachinery/pkg/labels"
 	vpa_types_v1beta2 "k8s.io/autoscaler/vertical-pod-autoscaler/pkg/apis/autoscaling.k8s.io/v1beta2"
+        "k8s.io/klog"
```
Comment (bskiba) [inline vertical-pod-autoscaler/pkg/target/composite_fetcher.go]: Please fix indent.

**Comment by Rajat-0** [comment]
@bskiba PTAL

**Comment by bskiba** [comment]
Verify is failing

**Comment by bskiba** [comment]
This is obsolete with #1992 
/close

**Pull request closed without merging**